### ISSUE TITLE
CamelCase?

### ISSUE THREAD

**Issue by Ekirapapaul** [Ekirapapaul/django-mpesa#19]
@muneneevans voting we should move all parameters, args etc to camel case. Example in the submit API move from phone_number to phoneNumber.

**Comment by muneneevans** [comment]
Yes. Considering it's json, camelCase might be implied. What about backward compatibility?
